app/logic/courseProposals.py

```diff
@@ -19,3 +22,19 @@ def getProposalData(user):
         "term":course.term,
         "status":course.status.status}
     return courseDict
+
+def deleteProposal(courseID):
+    """Deletes proposal of ID passed in. Removes foreign keys first.
+    Key Dependencies: QuestionNote, CourseQuestion, CourseParticipant,
+    CourseInstructor"""
+    course = Course.get(Course.id == courseID)
+    notes = (QuestionNote.select()
```

Comment:
Just change this query to a delete and execute it. No need to loop through each note.

Reply:
done!